Gems/Atom/Feature/Common/Assets/Passes/Mobile/Forward.pass

```diff
@@ -52,9 +52,9 @@
                     "LoadStoreAction": {
                         "ClearValue": {
                             "Value": [
-                                0.0,
-                                0.0,
-                                0.0,
+                                0.4,
+                                0.4,
+                                0.4,

```

Comment:
nbd but I recommend keeping a clear value of black otherwise it makes it difficult to draw stars on top additively

Reply:
I made it grey so it's consistent with the other render pipelines such as MainPipeline. 
Also, Carbonated project doesn't use a skybox but only uses this background with transparent cloud overlay. Keep it back won't look right unless they change their level data.